# ts-doc-external

Requires the existence of TSDOc commentss on all external objects and that such TSDoc comments do not include `@internal` or `@ignore` tags.

External items are defined as classes, interfaces, members of the previous two, or standalone functions that are exported from the main entrypoint to the package or are members of an export from the main entrypoint (defined recursively).

By default, the main entrypoint is assumed to be `src/index.ts`. However, if your package's main entrypoint is elsewhere, you'll need to specify so in your `.eslintrc` configuration file as follows (for example, if the entrypoint is `index.ts`):

This rule exists to ensure inclusion of external objects in documentation generated by tools such as [API Extractor](https://api-extractor.com/) and [TypeDoc](https://typedoc.org/).

## Examples

Assume all objects mentioned are not exported from the main entrypoint to the package.

### Good

```ts
/**
 * Other documentation
 */
class ExampleClass {
    /**
     * Other documentation
     */
    memberProperty: any,
    /**
     * Other documentation
     */
    memberFunction(): void {}
}
```

```ts
/**
 * Other documentation
 */
interface ExampleInterface {
  /**
   * Other documentation
   */
  memberProperty: any;
  /**
   * Other documentation
   */
  memberFunction: () => void;
}
```

```ts
/**
 * Other documentation
 */
function exampleFunction(): void {}
```

### Bad

```ts
class ExampleClass {
    /**
     * Other documentation
     */
    memberProperty: any,
    /**
     * Other documentation
     */
    memberFunction(): void {}
}
```

```ts
interface ExampleInterface {
  /**
   * Other documentation
   */
  memberProperty: any;
  /**
   * Other documentation
   */
  memberFunction: () => void;
}
```

```ts
function exampleFunction(): void {}
```

```ts
/**
 * Other documentation
 */
class ExampleClass {
    memberProperty: any,
    memberFunction(): void {}
}
```

```ts
/**
 * Other documentation
 * @internal
 */
class ExampleClass {
    /**
     * Other documentation
     * @ignore
     */
    memberProperty: any,
    /**
     * Other documentation
     */
    memberFunction(): void {}
}
```

## Source

Contrapositve of [ts-doc-internal].
