# ts-doc-internal

Requires any TSDoc comments on internal items to include either an `@internal` or an `@ignore` tag.

Internal items are defined as classes, interfaces, or standalone functions that are not exported from the main entrypoint to the package and are not members of any exports from the main entrypoint (defined recursively).

The main entrypoint is the same as the one used in `settings.main` for `ts-modules-only-named`.

This rule exists to prevent inclusion of internal items in documentation generated by tools such as [API Extractor](https://api-extractor.com/) and [TypeDoc](https://typedoc.org/).

## Examples

Assume all objects mentioned are not exported from the main entrypoint to the package.

### Good

```ts
/**
 * Other documentation
 * @internal
 */
class ExampleClass {}
```

```ts
/**
 * Other documentation
 * @ignore
 */
interface ExampleInterface {}
```

```ts
/**
 * Other documentation
 * @internal
 */
function exampleFunction(): void {}
```

### Bad

```ts
/**
 * Other documentation
 */
class ExampleClass {}
```

```ts
/**
 * Other documentation
 */
interface ExampleInterface {}
```

```ts
/**
 * Other documentation
 */
function exampleFunction(): void {}
```

## Source

Suggestion by [@chradek](https://github.com/chradek).
