# ts-package-json-author

Requires `author` in `package.json` to be set to `"Microsoft Corporation"`.

## Examples

### Good

```json
{
    "author": "Microsoft Corporation"
}
```

### Bad

```json
{
    "author": "Microsoft"
}
```

```json
{}
```

## [Source](https://azuresdkspecs.z5.web.core.windows.net/TypeScriptSpec.html#ts-package-json-author)
